policyengine_us/data/datasets/cps/cps.py

```diff
@@ -201,6 +201,7 @@ def add_personal_variables(cps: h5py.File, person: DataFrame) -> None:
     # "Is...blind or does...have serious difficulty seeing even when Wearing
     #  glasses?" 1 -> Yes
     cps["is_blind"] = person.PEDISEYE == 1
+    cps["is_ssi_disabled"] = person.DIS_HP == 1
```

Comment:
Though `DIS_HP` is limited to people age 15+. We may want an `or` condition over all disability flags: `PEDISDRS`, `PEDISEAR`, `PEDISEYE`, `PEDISOUT`, `PEDISPHY`, `PEDISREM`.
```suggestion
    # Who has a health problem or a disability which prevents work or which limits the kind or amount of work?
    cps["is_ssi_disabled"] = person.DIS_HP == 1
```